BitwardenShared/Core/Auth/Models/Response/IdentityTokenErrorModel.swift

```diff
@@ -44,13 +40,17 @@ struct IdentityTokenErrorModel: Codable {
 public struct AuthMethodsData: Codable, Equatable, Sendable {
     /// Key names used for encoding and decoding.
     enum CodingKeys: String, CodingKey {
+        case authenticator = "0"
         case email = "1"
         case duo = "2"
         case organizationDuo = "6"
         case yubikey = "3"
         case webAuthn = "7"
     }
 
+    /// Information for two factor authentication with Authenticator
+    let authenticator: Bool
```

Comment:
⛏️ In this case is whether it's enabled or not:
```suggestion
    /// Whether authenticator provider is enabled.
    let authenticator: Bool
```